disnake/permissions.py

```diff
@@ -157,10 +180,10 @@ def is_strict_superset(self, other: Permissions) -> bool:
         """Returns ``True`` if the permissions on self are a strict superset of those on other."""
         return self.is_superset(other) and self != other
 
-    __le__ = is_subset
-    __ge__ = is_superset
-    __lt__ = is_strict_subset
-    __gt__ = is_strict_superset
+    __le__ = is_subset  # type: ignore
+    __ge__ = is_superset  # type: ignore
+    __lt__ = is_strict_subset  # type: ignore
+    __gt__ = is_strict_superset  # type: ignore
```

Comment:
Could you maybe explain in a comment why the `# type: ignore` comments are used here?

Reply:
https://github.com/DisnakeDev/disnake/pull/605/commits/27bbb29b3d682d6a049a31871a0a2076968b0978